pkg/analysis/default-rules.yaml

```diff
@@ -52,7 +52,7 @@ Rules:
         Comment: "Exclude kube-system from analysis"
         Disabled: true
         Expression: |
-          has(subject.namespace) && (subject.namespace == "kube-system")
+          has(subject.name) && subject.name.startsWith('system:') && (subject.name != "system:serviceaccounts")
```

Comment:
The exclusion rule intent is avoid reporting on objects in kube-system -this change completely change this intent

Reply:
yes,i know ,but all serviceaccount  can give some privilege ,i found some customer use this

Reply:
@INT2ECALL in that case you can create a private analysis rules, tailored to your customer with the relevant changes and pass to the analysis run.

`rbac-tool analyze --config your-rules.yaml`

Reply:
@gadinaor-r7 cool ，thanks